src/gameObjects/HealthBar.js

```diff
@@ -0,0 +1,65 @@
+import Phaser from "phaser";
+
+const BLACK = 0x000000;
+const RED = 0xff0000;
+const WHITE = 0xffffff;
+
+const BAR_WIDTH = 64;
+const BAR_HEIGHT = 8;
+const BAR_BORDER_WIDTH = 2;
+const BAR_BORDER_WIDTH_DOUBLE = BAR_BORDER_WIDTH * 2;
+
+const OFFSET = {
+  X: -32,
+  Y: -64,
+};
+
+export default class HealthBar {
+  constructor(scene, x, y, value) {
+    this.bar = new Phaser.GameObjects.Graphics(scene);
+
+    this.x = x + OFFSET.X;
+    this.y = y + OFFSET.Y;
+    this.bar.depth = y;
+    this.value = value;
+    this.p = (BAR_WIDTH - BAR_BORDER_WIDTH_DOUBLE) / 100;
+
+    this.draw();
+
+    scene.add.existing(this.bar);
+  }
+
+  setPosition(x, y) {
+    this.x = x + OFFSET.X;
+    this.y = y + OFFSET.Y;
+    this.bar.depth = y;
+
+    this.draw();
+  }
+
+  draw() {
```

Comment:
It should be much easier to use rectangle instead:
https://github.com/photonstorm/phaser3-examples/blob/master/public/src/game%20objects/shapes/rectangle.js